apps/yggdrasil_web/web/controllers/character_controller.ex

```diff
@@ -0,0 +1,58 @@
+defmodule YggdrasilWeb.CharacterController do
+  use YggdrasilWeb.Web, :controller
+  use Guardian.Phoenix.Controller
+
+  alias Yggdrasil.Character
+
+  def index(conn, _params, user, _claims) do
+    user_id = user.id
```

Comment:
I would rather have the same API and enforce everything via permissions then to have separate APIs. Also I don't see the issue with `/characters` and `/characters?filter[user_id]=12` returning the same results. I would think in most cases things would be readable to those signed in anyway.